### DISCUSSION TITLE
Docker configuration

### DISCUSSION THREAD

**Discussion by yula-21** [Security-Onion-Solutions/securityonion#5187] [Unsupported Versions]
Hi,
Im trying to make adjustments to fit the needs of my project.
I would like to know if theres are configuration files for the different dockers on the machine.
I’ve read about docker-compose so I know there are configuration files for dockers I just cant find them.
If anyone has any idea I would love to hear it,
Thank you.

**Comment by weslambert** [comment]
The Docker configuration exists in each state directory, within the init.sls:

`/opt/so/saltstack/default/salt/`

(Copy files and make your modifications in `/opt/so/saltstack/local/salt`)
